app/assets/javascripts/components/ReportFilter.jsx

```diff
@@ -14,22 +30,23 @@ class ReportFilter extends React.Component {
       ? ReportFilter.getFilter('nt_rpm_threshold') : default_nt_rpm_threshold;
 
     const nt_zscore_start =
-      (nt_zscore_threshold.split(',').length > 0) ? parseInt(nt_zscore_threshold.split(',')[0], 10) : 0;
+      (nt_zscore_threshold.split(',').length > 0) ? parseInt(nt_zscore_threshold.split(',')[0], 10) :
+      this.lowest_nt_zscore;
     const nt_zscore_end =
       (nt_zscore_threshold.split(',').length > 1) ? parseInt(nt_zscore_threshold.split(',')[1], 10) :
-        this.props.highest_tax_counts.highest_nt_zscore;
+      this.highest_nt_zscore;
 
     const nt_rpm_start =
-      (nt_rpm_threshold.split(',').length > 0) ? parseInt(nt_rpm_threshold.split(',')[0], 10) : 0;
+      (nt_rpm_threshold.split(',').length > 0) ? parseInt(nt_rpm_threshold.split(',')[0], 10) :
+      this.lowest_nt_rpm;
     const nt_rpm_end =
       (nt_rpm_threshold.split(',').length > 1) ? parseInt(nt_rpm_threshold.split(',')[1], 10) :
-        this.props.highest_tax_counts.highest_nt_rpm;
-
+      this.highest_nt_rpm;
```

Comment:
There is a lot of repeated code here, when we go to 8 columns with 2 bounds each, it will likely be unmanageable.  Probably should generalize/parameterize so you don't have to repeat each line 16 times:  for each {highest, lowest} x {genus, species} x {nt, nr} x {zscore, rpm}.